public/components/integrations/home.tsx

```diff
@@ -65,7 +81,13 @@ export const Home = (props: HomeProps) => {
             exact
             path={'/available/:id/setup'}
             render={(routerProps) => (
-              <SetupIntegrationPage integration={decodeURIComponent(routerProps.match.params.id)} />
+              <SetupIntegrationPage
+                integration={decodeURIComponent(routerProps.match.params.id)}
+                dataSourceManagement={dataSourceManagement}
+                notifications={notifications}
```

Comment:
Do we need notifications here? Is it used anywhere?

Reply:
its used for the selector https://github.com/opensearch-project/dashboards-observability/pull/1992/files#diff-8aaf0df3a5da90db7fa5fdde34e79bb6df48fdfce0f0763272ee598266b76c2cR245